src/applications/claims-status/components/appeals-v2/AppealListItemV2.jsx

```diff
@@ -106,7 +106,7 @@ export default function AppealListItem({ appeal, name, external = false }) {
       {requestEvent && (
         <div className="card-status">
           <p>
-            <strong>Submitted on:</strong>{' '}
+            <strong>Received on:</strong>{' '}
```

Comment:
This claim details card is used when the following is true:
- claim type is a part of the appeal types that were passed in to YourClaimsPageV2.jsx
- useLighthouse = false
- useNewClaimCards is disabled.